packages/components/src/popover/stories/index.tsx

```diff
@@ -125,6 +127,108 @@ Default.args = {
 	),
 };
 
+export const Toolbar: ComponentStory< typeof Popover > = ( args ) => {
+	const [ isVisible, setIsVisible ] = useState( false );
+	const toggleVisible = () => {
+		setIsVisible( ( state ) => ! state );
+	};
+	const buttonRef = useRef< HTMLButtonElement | undefined >();
+	useEffect( () => {
+		buttonRef.current?.scrollIntoView?.( {
+			block: 'center',
+			inline: 'center',
+		} );
+	}, [] );
+
+	return (
+		<div
+			style={ {
+				width: '300vw',
+				height: '300vh',
+				display: 'flex',
+				alignItems: 'center',
+				justifyContent: 'center',
+			} }
+		>
+			<Button
+				variant="secondary"
+				onClick={ toggleVisible }
+				ref={ buttonRef }
+			>
+				Toggle Popover
+				{ isVisible && <Popover { ...args } /> }
+			</Button>
+		</div>
+	);
+};
+Toolbar.parameters = {
+	controls: {
+		exclude: [ 'variant' ],
+	},
```

Comment:
I understand the intention, but just as a matter of convention in our Storybook, we don't need to suppress controls like this stories. (Too much overhead compared to the upside 😆)

Reply:
Ah, I was just copying some other stories. Thanks for the info, I've removed those exclusions.